caffe2/onnx/backend.cc

```diff
@@ -515,6 +515,59 @@ Caffe2Ops Caffe2Backend::CreateConvPoolOpBase(
   return CommonOnnxNodeToCaffe2Ops(onnx_node, opset_version);
 }
 
+Caffe2Ops Caffe2Backend::CreatePadPool(OnnxNode* onnx_node, int opset_version) {
+  auto& node = onnx_node->node;
+  auto& attributes = onnx_node->attributes;
+  Caffe2Ops ret;
+  // Pad
+  bool padding = false;
+  auto pad_a = dummy_->NewDummyName();
+  if (attributes.HasAttribute("count_include_pad")) {
+    auto count_include_pad = attributes.get<int64_t>("count_include_pad", 0L);
+    if (count_include_pad == 1) {
+      std::string pad_name = opset_version < 2 ? "paddings" : "pads";
```

Comment:
There are some helper functions in https://github.com/pytorch/pytorch/blob/master/caffe2/utils/proto_utils.h

Reply:
Aren't those helper functions for c2 operators/arguments? I don't think we have c2 operators/arguments but onnx node/attributes yet upto this line. Please correct me if I am wrong.